spec/test_helper.rb

```diff
@@ -6,12 +6,14 @@
 RSpec.configure do |config|
   original_stderr = $stderr
   original_stdout = $stdout
-  config.before(:all) do
+
+  config.before(:all, silent_cli: true) do
```

Comment:
Sure, no param means `:each` option with `:silent_cli` tag. Specs are OK, Travis will build it soon.